middle_end/flambda/types/structures/row_like.rec.mli

```diff
@@ -22,6 +22,9 @@
    functor applications produce "undefined recursive module" errors. *)
 
 (* CR mshinwell: tidy up! *)
+(* CR gbury: Consider using Tag.Scannable.t instead of Tag.t in
+             For_blocks, and use another module to specifically handle float
+             arrays. *)
```

Comment:
All-float blocks (e.g. records all of whose elements are `float`s), not float arrays, which are already handled separately.

Reply:
fixed